### PR TITLE
Add colombia translation

### PR DESCRIPTION
**Pull request by samajammin**
## Description
Add colombia translation

## Related Issue
None

### CODE REVIEW

File: src/pages/local-grants/colombia/es.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Subvenciones Locales de Colombia
+description: Una línea de subvenciones de la Fundación Ethereum, específica para Colombia!
```
Comment (samajammin) [inline src/pages/local-grants/colombia/es.md:3]: @gilbertginsberg please review this title & description 😄

File: src/pages/local-grants/colombia/es.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+### **¿Cuál es el valor de las subvenciones locales?**
+
+- Los montos son decididos caso por caso.
+- Los beneficiarios tendrán la opción de recibir el valor en ETH, Dai, o Dólares Americanos (USD).
+
+## **🚀 ¿Cómo aplicar?** {#submit}
+
+_Completa el siguiente formulario de inscripción, para dejarnos saber sobre tu ideas relacionadas con Ethereum o tu proyecto ya existente; eso es todo. ¡Tras recibir tu aplicación, nos podremos en contacto vía correo electrónico!._
+
+<LocalGrantsForm wave="Colombia | 2020" />
```
Comment (samajammin) [inline src/pages/local-grants/colombia/es.md:80]: @gilbertginsberg note: this form is still in English. I'll need all the translated prompts / field names if we want to translate that as well.

**Comment by samajammin** [comment]
@gilbertginsberg FYI as is, esp.ethereum.foundation/colombia still redirects to the English page. Would you prefer it to resolve to the Spanish page now?

File: src/pages/local-grants/colombia/es.md (hunk 1 above)
Comment (gilbertginsberg) [reply src/pages/local-grants/colombia/es.md:3]: Let's go with: `Programa de Fondos Locales para Colombia`

File: src/pages/local-grants/colombia/es.md (hunk 2 above)
Comment (gilbertginsberg) [reply src/pages/local-grants/colombia/es.md:80]: I'll work on this. But we can deploy with the English version and then update later.

**Comment by gilbertginsberg** [comment]
@samajammin - yes, awesome. Let's have esp.ethereum.foundation/colombia redirect to the Spanish version. Thank you.

File: src/pages/local-grants/colombia/es.md (hunk 1 above)
Comment (gilbertginsberg) [reply src/pages/local-grants/colombia/es.md:3]: Description is solid

**Pull request merged**